.github/agents/README.md

```
@@ -9,18 +9,5 @@ Available templates
-Usage
-
-```js
-const prompt = fs.readFileSync('.github/agents/<template>.md', 'utf8')
-runSubagent({ prompt, description: '...' })
-```
-
-Notes
-- All templates use YAML frontmatter with `name` and `description`.
-- Prefer output in `apply_patch` format for code changes.
+- `general_purpose_agent.md` — General repo research, analysis, and small patch guidance.
+- `verification_agent.md` — Verification-only agent for build/test evidence and PASS/FAIL verdicts.
```

markdownlint-cli2 (0.23.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)